src/workloads/networking/CommitLatency.yml

```diff
@@ -63,214 +63,98 @@ Actors:
 
 
 
-# All of the below is really just the same actor, but as Genny reports statistics using the actor
-# name, we need a different actor instance for each configuration. All of the below run in their
-# own slot, nothing is concurrent.
-# Since Drop+Reload is in phases 0,1,-,3,4,-... then the test itself must always be in a slot at
-# 2,5,8... = 3n-1 = 3n+2
-- Name: CommitLatency_w0_local
+- Name: CommitLatency
```

Comment:
I got kinda dizzy trying to reason if this workload yaml ends up being equivalent to the existing one so I'm going to trust you there (or perhaps ask Henrik to take a peek).